src/ComparePage/ComparePage.tsx

```diff
@@ -0,0 +1,203 @@
+	displayContainer: () => ({
+		display: 'flex',
+		flexDirection: 'row',
+		justifyContent: 'space-between',
+	}),
+	searchContainer: () => ({
+		display: 'flex',
+		justifyContent: 'space-between',
+		alignItems: 'center',
+	}),
+})
+
+const ComparePage: FC = () => {
+	const { api, setTitle } = useContext(GlobalContext) // for page title and API access
+
+	const classes = useStyles(theme)
+	// List of users on server
+	const [users, setUsers] = useState<GetUsersEndpointResult[]>([])
+	// Users we are comparing
+	const [userA, setUserA] = useState<GetUsersEndpointResult>({
+		accountId: '0',
+		lastUpdated: 0,
+		masteryTotal: 0,
+		name: '0',
+		summonerId: '0',
+	})
+	const [userB, setUserB] = useState({
+		accountId: '0',
+		lastUpdated: 0,
+		masteryTotal: 0,
+		name: '0',
+		summonerId: '0',
+	})
+	// Const with setter for how many games stats to request from API
+	const [numberOfGames, setNumberOfGames] = useState({
+		count: 1,
+	})
+	// Const of A stats
+	const [userAStats, setUserAStats] = useState<any>()
+	// Const of B stats
+	const [userBStats, setUserBStats] = useState<any>()
+	// API call to get list of users
+	const fireGetUsers = useCallback(async () => {
+		const userList = await api.pingGetUsersEndpoint()
+		console.log('userlist ' + userList)
+		setUsers(userList)
+		setUserA(userList[0])
+		setUserB(userList[0])
+	}, [api])
+	// API call to get stats for a user
+	const fireGetStats = async (accountId: string): Promise<any> => {
+		const statsResult = await api.pingSummonerStatsEndpoint(
+			accountId,
+			numberOfGames.count
+		)
+		if (accountId == userA.accountId)
+			setUserAStats({
+				assistsAvg: statsResult.assistsAvg,
+				assistsTotal: statsResult.assistsTotal,
+				deathsAvg: statsResult.deathsAvg,
+				deathsTotal: statsResult.deathsTotal,
+				gamesCount: statsResult.gamesCount,
+				goldEarnedAvg: statsResult.goldEarnedAvg,
+				goldEarnedTotal: statsResult.goldEarnedTotal,
+				kDA: statsResult.kDA,
+				killsAvg: statsResult.killsAvg,
+				killsTotal: statsResult.killsTotal,
+				timePlayedAvg: statsResult.timePlayedAvg,
+				timePlayedTotal: statsResult.timePlayedTotal,
+				totalLosses: statsResult.totalLosses,
+				totalWins: statsResult.totalWins,
+				winPercentage: statsResult.winPercentage,
+			})
+		else if (accountId == userB.accountId) {
+			setUserBStats({
+				assistsAvg: statsResult.assistsAvg,
+				assistsTotal: statsResult.assistsTotal,
+				deathsAvg: statsResult.deathsAvg,
+				deathsTotal: statsResult.deathsTotal,
+				gamesCount: statsResult.gamesCount,
+				goldEarnedAvg: statsResult.goldEarnedAvg,
+				goldEarnedTotal: statsResult.goldEarnedTotal,
+				kDA: statsResult.kDA,
+				killsAvg: statsResult.killsAvg,
+				killsTotal: statsResult.killsTotal,
+				timePlayedAvg: statsResult.timePlayedAvg,
+				timePlayedTotal: statsResult.timePlayedTotal,
+				totalLosses: statsResult.totalLosses,
+				totalWins: statsResult.totalWins,
+				winPercentage: statsResult.winPercentage,
+			})
+		}
```

Comment:
We can de-dup some of this code; also, make sure to use triple equals

```suggestion
		)
		const result = {
			assistsAvg: statsResult.assistsAvg,
			assistsTotal: statsResult.assistsTotal,
			deathsAvg: statsResult.deathsAvg,
			deathsTotal: statsResult.deathsTotal,
			gamesCount: statsResult.gamesCount,
			goldEarnedAvg: statsResult.goldEarnedAvg,
			goldEarnedTotal: statsResult.goldEarnedTotal,
			kDA: statsResult.kDA,
			killsAvg: statsResult.killsAvg,
			killsTotal: statsResult.killsTotal,
			timePlayedAvg: statsResult.timePlayedAvg,
			timePlayedTotal: statsResult.timePlayedTotal,
			totalLosses: statsResult.totalLosses,
			totalWins: statsResult.totalWins,
			winPercentage: statsResult.winPercentage,
		}
		if (accountId === userA.accountId)
			setUserAStats(result)
		else if (accountId === userB.accountId) {
			setUserBStats(result)
		}
```